packages/container/lib/container.js

```diff
@@ -379,30 +379,24 @@ function areInjectionsDynamic(injections) {
   return !!injections._dynamic;
 }
 
-function buildInjections() /* container, ...injections */{
+function buildInjections(container, ...injectionsArray) {
   let hash = {};
+  let injections = Array.prototype.concat(...injectionsArray)
 
-  if (arguments.length > 1) {
-    let container = arguments[0];
-    let injections = [];
-    let injection;
-
-    for (let i = 1; i < arguments.length; i++) {
-      if (arguments[i]) {
```

Comment:
Shouldn't it be changed to push? [Benchmark for push vs concat.](http://jsbench.github.io/#1d1f32cc92a00df64e33d8b16aed4d82)